x-pack/plugin/security/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -45,3 +45,11 @@ grant codeBase "${codebase.netty-transport}" {
    // the bug says it only happened rarely, and that its fixed, but apparently it still happens rarely!
    permission java.util.PropertyPermission "sun.nio.ch.bugLevel", "write";
 };
+
+grant {
+  // needed for com.nimbusds.jose.shaded.gson.Gson.toJson
+  permission java.lang.RuntimePermission "accessDeclaredMembers";
+
+  // needed for com.nimbusds.oauth2.sdk.id.Identifier#value
+  permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+};
```

Comment:
It's more straightforward to read and understand policy files if there is just a **single** grant to each codebase (or non-codebase/global).

Reply:
Agreed. I am blocked trying to get this to work in the oidc-op-tests test fixture, so I thought maybe the codebase names were not being read. If I can get past whatever is blocking oidc-op-tests from applying a new policy, I can separate the grants by codebase again.

Reply:
I separated them by codebase again. I tried with and without codebase, no luck. The stacktrace is in the latest PR comment.